requirements/tests.txt

```diff
@@ -1,2 +1,3 @@
 flufl.lock
+packaging>=20
```

Comment:
setuptools has packaging distributed with it internally, do we want to use that instead of the new dependency? I haven't kept up with what the expected version comparison landscape is in Python lately, but a quick google seems like it is still quite dynamic.

Reply:
That's in `_vendor` and is private. `packaging` is the intended replacement for `distutils.version`.

Reply:
Thanks for the clarification!